code/game/turfs/turf.dm

```diff
@@ -168,12 +168,14 @@ GLOBAL_LIST_EMPTY(station_turfs)
 /turf/proc/multiz_turf_new(turf/T, dir)
 	SEND_SIGNAL(src, COMSIG_TURF_MULTIZ_NEW, T, dir)
 
-///returns if the turf has something dense inside it. if exclude_mobs is true, skips dense mobs like fat yoshi.
-/turf/proc/is_blocked_turf(exclude_mobs)
+///returns if the turf has something dense inside it. if exclude_mobs is true, skips dense mobs like fat yoshi. if exclude_object is true, it will exclude the excluded_object you sent through
+/turf/proc/is_blocked_turf(exclude_mobs, exclude_object = FALSE, excluded_object)
```

Comment:
Why not just send in the excluded object, and if it exists exclude it?

Reply:
Yer right, will change it in the next commit